experiment/measurer/measure_manager.py

```diff
@@ -617,7 +628,8 @@ def measure_snapshot_coverage(fuzzer: str, benchmark: str, trial_num: int,
         return models.Snapshot(time=this_time,
                                trial_id=trial_num,
                                edges_covered=regions_covered,
-                               fuzzer_stats=fuzzer_stats_data)
+                               fuzzer_stats=fuzzer_stats_data,
+                               crashes=[])
```

Comment:
no files changed in corpus archive which means no new crash files as well, use []

Reply:
ACK